packaging/etc/engine-config/engine-config.properties

```diff
@@ -550,6 +550,8 @@ LiveSnapshotPerformFreezeInEngine.type=Boolean
 LiveSnapshotFreezeTimeout.description=The live snapshot command without memory timeout in minutes
 LiveSnapshotFreezeTimeout.type=Integer
 LiveSnapshotFreezeTimeout.validValues=1..3000
+LiveStorageMigrationInitialSizeSupported.description=Enable initial size allocation when creating a snapshot during a Live Storage Migration.
+LiveStorageMigrationInitialSizeSupported.type=Boolean
```

Comment:
Why this need to be configurable? There is no reason to disable this feature. I think we need to remove the config value.

Reply:
I tried to work similar to `UseHybridBackup`, `IsIncrementalBackupSupported`, etc.
@bennyz What do you think, remove it and enable it for v4.7 and above?

Reply:
Also, just noticed, that there is an option to define it as "`ClientAccessLevel.Admin`". Worth doing so?

Reply:
this is a bug fix, there is no reason for toggle it off